buildSrc/src/main/java/org/robolectric/gradle/DeployedRoboJavaModulePlugin.kt

```diff
@@ -47,28 +45,6 @@ class DeployedRoboJavaModulePlugin : Plugin<Project> {
           from(javadocTask.map { it.destinationDir })
         }
 
-      project.tasks.withType<Javadoc> {
```

Comment:
I ran the Javadoc aggregation task locally to compare the output with the published 4.13 Javadoc.
I'll try the local publishing to Maven to see if the output is correct too.

Reply:
I've generated the Javadoc jar for 4.14-SNAPSHOT and compared it to the published 4.13 one, and there is a difference between the two (outside of the version change).
The `javadocOptions.bottom` is missing.

So I'm closing this PR for now, since it's breaking the artefacts. I'll look into grouping both configuration once everything is in Kotlin.